source/guide/manage-a-blog.html.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Manage a blog
+icon: book-open
+summary: 'How to customize your blog'
+---
+
+You can open your blog settings, by clicking on the "Edit" purple button in the top right part of its main page.
+
+# Adding a custom theme
+
+In your blog settings, you have a text box where you can enter any CSS.
+This CSS will be applied to your blog's main page, and to the articles published in this blog.
+You can do absolutely everything, expect using `url()` or `@import` to load external resources.
```

Comment:
are those the only dangerous ones?
does this mean i can't upload fonts and use them?

Reply:
Yes, but they are actually very hard to escape, so I'm trying another approach, where themes will be "approved" by admins, which will also allow for fonts and Co.

Reply:
(not relevant anymore, with the theming system that was finally adopted)